internal/cli/iam/projects/create.go

```diff
@@ -78,57 +87,85 @@ func (opts *CreateOpts) newCreateProjectOptions() *atlas.CreateProjectOptions {
 }
 
 func (opts *CreateOpts) validateOwnerID() error {
-	if opts.projectOwnerID == "" {
+	if opts.projectOwnerID == "" || opts.serviceVersion == nil {
 		return nil
 	}
 
-	if config.Service() != config.OpsManagerService {
+	constrain, err := semver.NewConstraint(">= 6.0")
+	if err != nil {
+		return err
+	}
+
+	if !constrain.Check(opts.serviceVersion) {
+		return fmt.Errorf("%s is available only for Atlas, Cloud Manager and Ops Manager >= 6.0", flag.OwnerID)
+	}
+
+	return nil
+}
+
+func (opts *CreateOpts) validateWithoutDefaultAlertSettings() error {
+	if !opts.withoutDefaultAlertSettings || opts.serviceVersion == nil {
 		return nil
 	}
 
-	v, err := opts.store.ServiceVersion()
+	constrain, err := semver.NewConstraint(">= 6.0")
 	if err != nil {
 		return err
 	}
 
-	sv, err := cli.ParseServiceVersion(v)
+	if !constrain.Check(opts.serviceVersion) {
+		return fmt.Errorf("%s is available only for Atlas, Cloud Manager and Ops Manager >= 6.0", flag.WithoutDefaultAlertSettings)
+	}
+
+	return nil
+}
+
+func (opts *CreateOpts) newServiceVersion() error {
+	if config.Service() != config.OpsManagerService {
+		return nil
+	}
+	v, err := opts.store.ServiceVersion()
 	if err != nil {
 		return err
 	}
 
-	constrain, _ := semver.NewConstraint(">= 6.0")
-
-	if !constrain.Check(sv) {
-		return fmt.Errorf("%s is available only for Atlas, Cloud Manager and Ops Manager >= 6.0", flag.OwnerID)
+	sv, err := cli.ParseServiceVersion(v)
+	if err != nil {
+		return err
 	}
 
+	opts.serviceVersion = sv
 	return nil
 }
 
-// mongocli iam project(s) create <name> [--orgId orgId] [--ownerID ownerID].
+// mongocli iam project(s) create <name> [--orgId orgId] [--ownerID ownerID] [--withoutDefaultAlertSettings].
 func CreateBuilder() *cobra.Command {
 	opts := &CreateOpts{}
 	opts.Template = atlasCreateTemplate
 	cmd := &cobra.Command{
-		Use:   "create <name>",
+		Use:   "create <projectName>",
 		Short: "Create a project.",
 		Args:  require.ExactArgs(1),
+		Annotations: map[string]string{
+			"projectNameDesc": "projectName",
+			"requiredArgs":    "Name of the project.",
```

Comment:
copy/pasta?
```suggestion
			"args": "projectName",
			"requiredArgs":    "projectName",
			"projectNameDesc": "Name of the project.",
```

Reply:
updated

Reply:
still not correct

Reply:
I don't understand 🤔 . Here you are suggesting to add args too, which is what I did

Reply:
<img width="1322" alt="Screenshot 2022-01-10 at 15 00 43" src="https://user-images.githubusercontent.com/5663078/148787458-ee1de72c-fe8c-41bc-bda4-bd73bf54c5c1.png">

Reply:
ah I see now

Reply:
updated